sections/custom_fields.md

```diff
@@ -0,0 +1,37 @@
+# Custom fields (Beta)
+
+Custom fields are enabled for PROJECT, PHASE, TIME_REG and TASK, for setting custom field value for an entity the custom field for that entity type have to be defined in the admin UI.
```

Comment:
"enabled" - shouldn't that maybe be "available"?
I think a "." is needed after "TASK".